docker/checkout.bash

```diff
@@ -107,3 +107,7 @@ cd V2X-Hub
 git config core.sparsecheckout true
 git sparse-checkout init
 git sparse-checkout set ext/ccserver
+sudo apt-get update && sudo apt-get install -y libdlib-dev
+
+cd ${dir}/src
+sudo bash multiple_object_tracking/scripts/install_dependencies.sh
```

Comment:
Since this script installs external dependencies, I think it makes sense to keep it in `docker/install.sh`, though perhaps eventually we can consider a separate `docker/install_external_depencies.sh` script, so that it can be called directly from our CI workflow. Can we move this back to `docker/install.sh` and add the command as a GitHub actions step?